README.rst

```diff
@@ -23,8 +23,6 @@ Disclosure
 -  This code has not been through a third party security audit.
 -  I’ve successfully tested this locally with ``pypy-5.4.1``. TravisCI
    has confirmed this doesn’t work with their version.
--  From their logs:
-   ``RuntimeError: PyNaCl is not compatible with PyPy < 2.6. Please upgrade PyPy to use this library.``
 
```

Comment:
Yeah I suppose "support" is the wrong word. I meant just something saying that DCN doesn't use PyNacl anymore. So that anybody updating from the current version can remove it if they don't need it for anything else.